### PR TITLE
Jpa

### PR DESCRIPTION
**Pull request by alexgutierrez-wizeline**

### CODE REVIEW

File: 5/Core/LearningJava.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Wizeline-GerardoRuiz) [inline 5/Core/LearningJava.iml]: Please remove this file from this folder, it's not necessary

File: 5/JPA/LearningJava/HELP.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Getting Started
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/HELP.md]: Please review this file as well as it's not necessary

File: 5/JPA/LearningJava/LearningJava.iml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/LearningJava.iml]: Same as above

File: 5/JPA/LearningJava/file.txt [hunk 4]
```diff
@@ -0,0 +1,30 @@
+alex, thenewepoch
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/file.txt]: Same as above

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+import java.time.Instant;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.function.Function;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+
+import org.json.JSONArray;
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java]: Please remove all unused imports in this class

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+import com.wizeline.maven.learningjava.model.ResponseDTO;
+import com.wizeline.maven.learningjava.model.UserDTO;
+import com.wizeline.maven.learningjava.service.BankAccountService;
+import com.wizeline.maven.learningjava.service.BankAccountServiceImpl;
+import com.wizeline.maven.learningjava.service.UserService;
+import com.wizeline.maven.learningjava.service.UserServiceImpl;
+import com.wizeline.maven.learningjava.utils.exceptions.ExcepcionGenerica;
+
+@SpringBootApplication
+public class LearningJavaApplication extends Thread {
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java]: Remove Thread inheritance

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+	private static final Logger LOGGER = Logger.getLogger(LearningJavaApplication.class.getName());
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java]: Remove logger and respective call

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/repository/UserRepository.java [hunk 8]
```diff
@@ -0,0 +1,16 @@
+/**
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/repository/UserRepository.java]: Remove this interface, let's use the implementation directly

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/utils/CommonServices.java [hunk 9]
```diff
@@ -0,0 +1,30 @@
+/*
```
Comment (Wizeline-GerardoRuiz) [inline 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/utils/CommonServices.java:1]: Please remove this class as it's not necessary

**Review by Wizeline-GerardoRuiz — CHANGES_REQUESTED** [review]

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/utils/CommonServices.java (hunk 9 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/utils/CommonServices.java:1]: This class contains one common method that is used for login

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java (hunk 7 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java]: ok

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/repository/UserRepository.java (hunk 8 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/repository/UserRepository.java]: ok

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java (hunk 6 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java]: ok

File: 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java (hunk 5 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/LearningJavaApplication.java]: ok

File: 5/Core/LearningJava.iml (hunk 1 above)
Comment (alexgutierrez-wizeline) [reply 5/Core/LearningJava.iml]: ok

File: 5/JPA/LearningJava/HELP.md (hunk 2 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/HELP.md]: ok

File: 5/JPA/LearningJava/LearningJava.iml (hunk 3 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/LearningJava.iml]: ok

File: 5/JPA/LearningJava/file.txt (hunk 4 above)
Comment (alexgutierrez-wizeline) [reply 5/JPA/LearningJava/file.txt]: ok

**Review by Wizeline-GerardoRuiz — APPROVED** [review]
LGTM 👍

**Pull request merged**